docs/src/examples.md

```diff
@@ -0,0 +1,125 @@
+    
+With flexibility and composability as major considerations in the design of
+StyledStrings, it is easy to describe the capabilities of this system while
+failing to actually convey what it can accomplish. With this in mind, examples
+can be particularly useful to show how it can be used.
+
+A styled string can be constructed manually, but the [`styled"..."`](@ref
+@styled_str) literal is almost always a nicer option. We can see what a manual construction would involve by extracting the string and annotation parts of a [`AnnotatedString`](@ref Base.AnnotatedString).
+
+```@repl examples
+using StyledStrings
+str = styled"{yellow:hello} {blue:there}"
+(String(str), Base.annotations(str))
+```
+
+Most of the examples here will show [`AnnotatedString`](@ref Base.AnnotatedString) in a terminal/REPL context, however they can be trivially adapted to produce HTML with `show(::IO, ::MIME"text/plain", ::AnnotatedString)`, and other packages wanting do deal with styled content in other contexts will likely find integration with StyledStrings worthwhile.
+
+As an end-user or a package-author, adding color is one of if not the most frequent application of styling. The default set of colours has the eight ANSI colors, and if we're to be brutally honest, `black`/`white` are just shades, leaving us with:
+
+```@repl examples
+styled"{red:■} {green:■} {yellow:■} {blue:■} {magenta:■} {cyan:■}"
+```
+
+along with "bright"  variants
+
+```@repl examples
+styled"{bright_red:■} {bright_green:■} {bright_yellow:■} {bright_blue:■} {bright_magenta:■} {bright_cyan:■}"
+```
+
+This seems somewhat limited, because it is. This is only the _default_ set of
+colours though. It is important to note that the way the colour `red` is
+implemented is by having it name a `Face(foreground=:red)` value. The ANSI
+printer knows to handle the ANSI named colours specially, but you can create
+more "named colours" simply by adding new faces.
+
+```@repl examples
+StyledStrings.loadface!(:orange => StyledStrings.Face(foreground = 0xFF7700))
+styled"{orange:this is orange text}"
+```
+
+The fact that named colors are implemented this way also allows for other nice
+conveniences. For example, if you waned a more subtle version of the warning
+styling, you could print text with the underline colour set to the warning
+foreground color.
+
+```@repl examples
+styled"{(underline=warning):this is some minor/slight warning text}"
+styled"A very major {(fg=error,bg=warning),bold:warning!}"
+```
+
+Should you want to use a particular colour just once, the `foreground`/`fg` and `background`/`bg` inline face attributes can be set to hex codes.
+
+```@repl examples
+styled"{(fg=#4063d8):ju}{(fg=#389826):l}{(fg=#cb3c33):i}{(fg=#9558b2):a}"
+```
+
+It is recommended that package authors create faces with a focus on the semantic
+meaning they wish to impart, and then consider what styling suits. For example,
```

Comment:
impart? or import

Reply:
It is meant to be impart here, but thanks for checking